### ISSUE TITLE
Deprecate this plugin

### ISSUE THREAD

**Issue by TheMarex** [perliedman/lrm-mapbox#7]
Since OSRM 5.0 support landed in the new LRM version, it automatically supports Mapbox Directions v5 and above as well (it is the same API just with text instructions).

I would probably help to put a deprecation disclaimer here since the v4 API does not expose all the new nice things like advanced guidance instructions. :-)

**Comment by perliedman** [comment]
Alright, I haven't kept up with the changes on the Mapbox side, but this sounds very good.

I had to add the ability to specify a `access_token` parameter to LRM's OSRMv1 class, but that's no issue.

However, I ran into what appears to be a difference between the APIs regarding the `hints` parameter. This OSRM request works:

https://router.project-osrm.org/route/v1/driving/11.94,57.74;11.949,57.6792?overview=false&alternatives=true&steps=true&hints=; (although it gives a timeout at the moment)

The corresponding Mapbox request gives an error:

https://api.mapbox.com/directions/v5/mapbox/driving/11.94,57.74;11.949,57.6792?overview=false&alternatives=true&steps=true&hints=;&access_token=pk.eyJ1IjoibGllZG1hbiIsImEiOiIyZmM4MDYxMTdiMTYwOTIyYzg1YmQxMGM3YTRhNjY0NCJ9.gJ6ITQll8ikdsYYtNtAjkg

I seem to remember OSRM having the same problem a couple of versions ago, might this be an old bug that still lives in the Mapbox API?
